### ISSUE TITLE
why did you add the testset to the training data?

### ISSUE THREAD

**Issue by languandong** [pengfeiliHEU/M2I2#10]
![Snipaste_2023-11-30_23-27-14](https://github.com/pengfeiliHEU/M2I2/assets/44127403/c3b844a2-b7a8-4d5e-bbad-1224dddbceb4)
why did you add the testset to the training data?
This is different from what we did before.
Is this reasonable?

**Comment by pengfeiliHEU** [comment]
![image](https://github.com/pengfeiliHEU/M2I2/assets/102028558/b6ed6088-cc72-4ce1-a085-4aa2573a24ef)
Please take a look here, it's incorrect to put the test set into the train set, there's no reason for this, and I don't know why the code is doing it this way. I checked the repository, and it's not there.

**Comment by languandong** [comment]
> ![image](https://private-user-images.githubusercontent.com/102028558/286990583-b6ed6088-cc72-4ce1-a085-4aa2573a24ef.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEzOTM3NjQsIm5iZiI6MTcwMTM5MzQ2NCwicGF0aCI6Ii8xMDIwMjg1NTgvMjg2OTkwNTgzLWI2ZWQ2MDg4LWNjNzItNGNlMS1hMDg1LTRhYTI1NzNhMjRlZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIwMVQwMTE3NDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02NWJmMTg3NmM3NzJkZGE3NDM5YTBlNGQ4OThiMDU4MDYxZmU5MmQwNTFhZDc3MTA1OWUxZGQxOTBiNDgxMGM3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.g2DXHbToCB0Xn0MVqw-4fDgyA7uHV9uLQCnjFM5vG6Q) Please take a look here, it's incorrect to put the test set into the train set, there's no reason for this, and I don't know why the code is doing it this way. I checked the repository, and it's not there.

https://github.com/pengfeiliHEU/M2I2/issues/10#issuecomment-1834061712
OK，that is in the last version really.
